pywebpush/__main__.py

```diff
@@ -0,0 +1,54 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
It's quite strange to have a ``__main__.py`` file, if this is a CLI (it appears to be), calling it cli.py is fine. Double underscore's are generally reserved for ``__init__.py``, or special Python constructs.